src/domain/domain.ts

```diff
@@ -212,18 +223,19 @@ export class DomainManager implements EntityManager<Domain, AddDomain> {
     throwOnCollision = true,
   ): Promise<AddDomain[]> {
     domains = await DomainManager.addManySchema.parseAsync(domains)
+    if (!throwOnCollision) return domains
 
     const currentDomains = await this.getAll()
     const currentDomainSet = new Set<string>(currentDomains.map((d) => d.name))
 
-    if (!throwOnCollision) {
+    /* if (!throwOnCollision) {
```

Comment:
The purpose of this is not signing domain messages for already existing domains. Why are we removing it?